includes/extra_configures/set_time_zone.php

```diff
@@ -8,21 +8,19 @@
 /*
  * Set time zone
 */
-  // put your timezone here. Refer to http://www.php.net/manual/en/timezones.php
-  $TZ = '';  // eg: 'Europe/Oslo'
+// put your timezone here. Refer to http://www.php.net/manual/en/timezones.php
+$TZ = '';  // eg: 'Europe/Oslo'
 
-
-
-  /**
-   * MAKE NO CHANGES BELOW THIS LINE
-   *
-   * The following will take the timezone you specified above and apply it in your store.
-   * If you didn't specify one, it will try to use the setting from your server's PHP configuration
-   */
-  if ($TZ == '') {
+/**
+* MAKE NO CHANGES BELOW THIS LINE
+*
+* The following will take the timezone you specified above and apply it in your store.
+* If you didn't specify one, it will try to use the setting from your server's PHP configuration
+*/
+if ($TZ === '') {
```

Comment:
In this file, with `$TZ`, I'd prefer to use `empty()` checking instead of `===` checking.

Reply:
How about just reverting to `==`?  Why waste a function call for a known string?